src/scrabble.js

```diff
@@ -1,5 +1,35 @@
-function scrabble() {
-  // write code here
+function scrabble(word) {
+  if (!word) return 0;
```

Comment:
I'm not sure this is 100% for stopping weird input – for example, it would let weird characters pass or (probably) arrays, objects… I'd check for the word containing only allowed characters to make sure

Reply:
Ok so basically what I meant by this – the current logic is "if the opposite of the input is true, return 0". But as you have a very specific profile on what kind of input you want to allow, I'd take an approach that checks more selectively – it should throw out everything that is not a word containing nothing but allowed characters, which are: A-Z, a-z, and the different types of brakets: [ ] { }.

here's a hint on how you check for contents of a string when you don't know exactly what the string will be: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match

Reply:
Regular expressions can be a lot at first but it's best explained at defining a pattern that you look for instead of exactly knowing what you want. If you want to see the described logic in code, have a look [at my file at line 11](https://github.com/boolean-uk/js-scrabble-challenge/pull/152/files#diff-df48340736af99ff21a33d27c2cdc9190750f76abe510659a40943f897880624)